backend/svirt.pm

```diff
@@ -312,7 +314,7 @@ sub get_wait_still_screen_on_here_doc_input ($self, $args) {
     # by comparing the screen and checking that nothing else will write on it.
     # So if the here-document input is really slow, we hope the wait_still_screen
     # takes even longer.
-    ($bmwqemu::vars{VIRSH_VMM_FAMILY} // '') =~ qr/^hyperv|vmware$/ ? 1 : 0;
+    _vmm_family() =~ qr/^hyperv|vmware$/ ? 1 : 0;
```

Comment:
Not related to the change, but I guess this regex should rather be: `qr/^(hyperv|vmware)$/` (otherwise it would allow `hypervsomething` or `somethingvmware`)

Reply:
good catch. I added an according commit to this PR